source/bindbc/freeimage/binddynamic.d

```diff
@@ -846,14 +846,23 @@ FISupport loadFreeImage(const(char)* libName)
     lib.bindSymbol_stdcall(FreeImage_GetMetadataCount, "FreeImage_GetMetadataCount");
     lib.bindSymbol_stdcall(FreeImage_CloneMetadata, "FreeImage_CloneMetadata");
     lib.bindSymbol_stdcall(FreeImage_TagToString, "FreeImage_TagToString");
-    lib.bindSymbol_stdcall(FreeImage_JPEGTransform, "FreeImage_JPEGTransform");
-    lib.bindSymbol_stdcall(FreeImage_JPEGTransformU, "FreeImage_JPEGTransformU");
-    lib.bindSymbol_stdcall(FreeImage_JPEGCrop, "FreeImage_JPEGCrop");
-    lib.bindSymbol_stdcall(FreeImage_JPEGCropU, "FreeImage_JPEGCropU");
-    lib.bindSymbol_stdcall(FreeImage_JPEGTransformFromHandle, "FreeImage_JPEGTransformFromHandle");
-    lib.bindSymbol_stdcall(FreeImage_JPEGTransformCombined, "FreeImage_JPEGTransformCombined");
-    lib.bindSymbol_stdcall(FreeImage_JPEGTransformCombinedU, "FreeImage_JPEGTransformCombinedU");
-    lib.bindSymbol_stdcall(FreeImage_JPEGTransformCombinedFromMemory, "FreeImage_JPEGTransformCombinedFromMemory");
+    // The JPEGTransform functions are deliberately disabled in Debian build
+    // of FreeImage, since they require usage of the vendored copy of libjpeg.
+    // Add the following in dub.json:
+    //    "versions": ["Debian"]
+    // or in dub.sdl:
+    //    versions "Debian"
+    version(Debian) {
```

Comment:
Rather than `Debian`, I'd prefer `BindBC_Debian` to avoid potential clashes and remain consistent with other BindBC versions.